cves/CVE-2016-2327.yml

```diff
@@ -131,7 +133,10 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    The lifetime spanned about 6 months, during which the vulnerable code was
+    left unchanged. The FFmpeg project, however, experienced large growth, with
+    over 8600 commits occurring during the lifespan of this vulnerability.
```

Comment:
Good. The file was really completely unchanged between the two, or was it just the region of code? What about who introduced it vs who fixed it?